packages/web-components/fast-components/src/switch/switch.styles.ts

```diff
@@ -156,5 +163,12 @@ export const SwitchStyles = css`
             background: ${SystemColors.Field};
             border-color: ${SystemColors.GrayText};
         }
+
+        :host(.disabled) .label,
+        :host(.disabled) .unchecked-message,
+        :host(.disabled.checked) .checked-message {
+            forced-color-adjust: none;
```

Comment:
I know we've typically colored the label of a disabled control as disabled but I want to point out that is a deviation from the default behavior of form elements and labels. See https://codepen.io/rice119/pen/LYpNGVY w/ high-contrast. The text does not change color, only the checkbox. Have we gotten guidance one way or the other on this?

Reply:
We can certainly get feedback on this - I think it's fair. I'd suggest we either keep this behavior for now and remove later with an issue, or remove here and track whether we should add this behavior. @nicholasrice do you have a preference one way or another while we land this?

Reply:
My general preference is platform alignment unless we have good reasons to deviate.

Reply:
This is generally what Windows does throughout the system, setting color to both input and label.  Not saying we have to follow what Windows is doing, but if I high contrast is Windows specific, then maybe we match it.

Reply:
@khamudom let's follow the platform here and remove the change to label text color when disabled

Reply:
screenshot also updated